app/trip/graphql/query/tripDetails.graphql

```diff
@@ -0,0 +1,42 @@
+query tripDetails($id: BigInt!) {
+  tripsCollection(filter: { id: { eq: $id } }) {
```

Comment:
I used $id not $UUID. Is that okay? Or should I use UUID?

Reply:
We'll eliminate SERIAL id and would rather use UUID down the road !
See: https://github.com/tabi-memo/frontend/pull/11#discussion_r1414762301

Reply:
Let tell me why did you use id rather than UUID?
As yoshi mentioned, UUID is better than id in terms of backend.

Reply:
Thank you! I changed to UUID 
https://github.com/tabi-memo/frontend/pull/17/commits/c86cda7f0f0a9112492a31b1f66cbee4897ba5a4